From 8bdd6ab90c26127ea68a8995ffbb23fb86da8247 Mon Sep 17 00:00:00 2001
From: Alain Volmat <alain.volmat@foss.st.com>
Date: Wed, 16 Oct 2024 13:13:08 +0200
Subject: [PATCH] arm64: dts: st: fix nodes for imx335/csi/dcmipp in
 stm32mp215f-dk

Use a generic name for the camera node (cf
https://devicetree-specification.readthedocs.io/en/latest/chapter2-devicetree-basics.html#generic-names-recommendation).
Avoid useless status = "okay" in the node since it is never disabled.

Change-Id: Ib742d430fa88b313b905de10ca0c743dec4c612d
Signed-off-by: Alain Volmat <alain.volmat@foss.st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/411736
Domain-Review: Philippe CORNU <philippe.cornu@foss.st.com>
Reviewed-by: Alain VOLMAT <alain.volmat@st.com>
Reviewed-by: Philippe CORNU <philippe.cornu@foss.st.com>
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
Tested-by: Alain VOLMAT <alain.volmat@st.com>
---
 arch/arm64/boot/dts/st/stm32mp215f-dk.dts | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/arch/arm64/boot/dts/st/stm32mp215f-dk.dts b/arch/arm64/boot/dts/st/stm32mp215f-dk.dts
index 6502d97cb7ed..a8da85304d79 100644
--- a/arch/arm64/boot/dts/st/stm32mp215f-dk.dts
+++ b/arch/arm64/boot/dts/st/stm32mp215f-dk.dts
@@ -311,13 +311,12 @@ goodix: goodix-ts@5d {
 		status = "okay" ;
 	};
 
-	imx335: imx335@1a {
+	imx335: camera@1a {
 		compatible = "sony,imx335";
 		reg = <0x1a>;
 		clocks = <&clk_ext_camera>;
 		reset-gpios = <&gpiod 5 (GPIO_ACTIVE_LOW | GPIO_PUSH_PULL)>;
 		powerdown-gpios = <&gpiod 0 (GPIO_ACTIVE_HIGH | GPIO_PUSH_PULL)>;
-		status = "okay";
 
 		port {
 			imx335_ep: endpoint {
-- 
2.50.1

